app.yaml

```diff
@@ -147,3 +147,22 @@ libraries:
 # running the server in the dev environment.
 env_variables:
   MINIFICATION: false
+
+skip_files:
```

Comment:
OK, but we should not do that in the main codebase unless there's a good reason to. I see lots of stuff that I don't recognize (fabfile.py, testrunner.py, RCS, ...) -- why are these there?

Consider removing this stanza from this PR.